README.md

```diff
@@ -42,6 +42,14 @@ php artisan make:model-auditlog "\App\User"
 
 Replace `\App\User` with your own model name. Model / table options can be tweaked in the config file.
 
+If you need to ignore specific fields on your model, extend the `getAuditLogIgnoredFields()` method and return an array of fields.
```

Comment:
Maybe include an example with a more complex set of steps to highlight the usefulness of this being a method vs a const array